### ISSUE TITLE
[BUG]: ./gradlew: not found error when trying to build and run docker compose on Windows

### ISSUE THREAD

**Issue by dhruv2196** [Netflix/conductor#3741]
## What are you missing in the docs
When we try to build docker image for conductor server on Windows, it throws -
> [conductor:server builder 4/4] RUN ./gradlew build -x test --stacktrace:
#24 0.398 /bin/sh: 1: ./gradlew: not found
So I found a solution to do that and I have added steps for the same below.
#### ./gradlew: not found when running on Windows 
1. Switch the line endings to "LF" from "CLRF" in gradlew file. 
In IntelliJ - From the main menu, select File | File Properties | Line Separators, and then select LF - Unix and MacOS(\n).
In Eclipse - Go to Window | Preferences | General | Workspace | New Text file line delimiter -> Other -> Unix
2. From the `docker` directory,
    ```
    docker build --no-cache -t conductor:server -f server/Dockerfile ../

**Comment by dhruv2196** [comment]
I have added a solution for it. Please let me know if I can assign it to myself and get it merged.

**Comment by dhruv2196** [comment]
I found another similar issue that when we are trying to build docker compose on windows. I was facing an error ./gradlew: not found.
I fixed it by changing the line seperator in gradlew so docker build was able to built it successfully but when I tried to run the server it was again throwing some exceptions. Reason for this issue is that when we clone code in windows machine it by default uses CRLF as line separator and due to this all scripts have CRLF line separators as EOL and they broke.
